doc/source/advanced.rst

```diff
@@ -538,18 +530,6 @@ an exception. Here is a concrete example to illustrate this:
    reordered = idx[[1, 0, 3, 2]]
    reordered.lexsort_depth
 
-   s = pd.Series(np.random.randn(4), index=reordered)
-   s.ix['a':'a']
-
-However:
-
-::
-
-   >>> s.ix[('a', 'b'):('b', 'a')]
-   Traceback (most recent call last)
-        ...
-   KeyError: Key length (3) was greater than MultiIndex lexsort depth (2)
-
```

Comment:
I added it back (in the context of the slightly changed example). lmk if more clear.
cc @MaximilianR